modules/theme/src/themes/default/elements/divider.overrides

```diff
@@ -11,4 +11,8 @@
         margin-top: @dividerCompactMargin;
         margin-bottom: @dividerCompactMargin;
     }
+
+    &.fido-list-divider {
```

Comment:
Shall we make this genric? Divider already supports few variations `2x, 3x` etc.

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/2815/commits/f90c3ca553a1f5b6bebde334933133841e096a00